### DISCUSSION TITLE
How to tell which workflow contains precompiled parsers?

### DISCUSSION THREAD

**Discussion by elan-hwang** [nvim-treesitter/nvim-treesitter#2419] [Q&A] [answered]
I have some problem compiling the parsers on my machine on Windows, so I need to download precompiled parsers. However, it is very difficult to tell which workflow actually contains artifacts:
https://github.com/nvim-treesitter/nvim-treesitter/actions/workflows/check-query-files-and-compilation.yml

**Comment by theHamsta** [comment, marked as answer]
We recently removed the pre-compiled parsers from the CI with the goal to add a proper release workflow which can be easier to discover in future.
